tests/platform/cloudfoundry/test_live_discovery.py

```
@@ -47,19 +47,22 @@ def clone_helm_chart_repo():
             missing.append("CLOUDFOUNDRY_FILES_PATH")
         if not cf_host:
             missing.append("CF_HOST")
-        if not cf_user:
-            missing.append("CF_USER")
+        if not cf_ssh_user:
+            missing.append("CF_SSH_USER")
         if not cf_remote_config_path:
             missing.append("CF_REMOTE_CONFIG_PATH")
+        if not cf_admin_password:
+            missing.append("CF_ADMIN_PASSWORD")
         raise Exception(f"Required environment variables not set: {', '.join(missing)}")
```

Ruff (0.14.11)

Create your own exception

(TRY002)